python/chemiscope/input.py

```diff
@@ -165,6 +208,46 @@ def create_input(
         for name, value in properties.items():
             data["properties"].update(_linearize(name, value, n_structures, n_atoms))
 
+    if parameters is not None:
+        if not isinstance(parameters, dict):
+            raise ValueError(
+                f"expecting parameters to be a of type 'dict' not type {type(parameters)}"
+            )
+        data["parameters"] = {}
+        for key in parameters:
+            param = {}
+            if not isinstance(parameters[key], dict):
+                raise ValueError(
+                    f"expecting parameter {key} to be of type 'dict' not type {type(parameters[key])}"
+                )
+            if not isinstance(parameters[key]["values"], np.ndarray):
+                raise ValueError(
+                    f"expecting the 'values' of parameter {key} to be of type 'numpy.ndarray' not {type(parameters[key]['values'])}"
+                )
```

Comment:
I would allow lists as well here, so how about converting this first with `np.asarray(parameters[key]["values"])`?

Reply:
I agree